src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Wizard.vue

```diff
@@ -68,11 +101,19 @@ export default defineComponent({
       alertStore,
     };
   },
+  computed: {
```

Comment:
This would be better if it existed in the pinia store `wizard.ts`

Reply:
You should be able to put it in `getters` of the pinia store

Reply:
Implemented the changes.